unlock-app/src/components/interface/user-account/PaymentDetails.tsx

```diff
@@ -12,17 +12,18 @@ import { Input, Label, Select, Button } from '../checkout/FormStyles'
 import { SectionHeader } from './styles'
 import configure from '../../../config'
 import { countries } from '../../../utils/countries'
-import { useProvider } from '../../../hooks/useProvider'
-import { StorageServiceContext } from '../../../utils/withStorageService'
-import { StorageService } from '../../../services/storageService'
 
 const { stripeApiKey } = configure()
 const stripePromise = loadStripe(stripeApiKey)
 
-export const PaymentDetails = () => {
+interface PaymentDetailsProps {
+  saveCard: (stripeToken: string) => any
```

Comment:
There are a few references to `saveCard` here, is this in support of the feature to delete cards or is this from some unrelated work?

If it is related to this feature, i'd like to suggest renaming this to better reflect the usage

Reply:
Sorry this is irrelevant to this PR :/ I did `git checkout` from a branch which had this.